extensions/dicom-pdf/src/ConnectedOHIFDicomPDFViewer.js

```diff
@@ -0,0 +1,33 @@
+import OHIF from '@ohif/core';
+import { connect } from 'react-redux';
+import OHIFDicomPDFViewport from './OHIFDicomPDFViewport';
+
+const { setViewportActive } = OHIF.redux.actions;
+
+const mapStateToProps = (state, ownProps) => {
+  const { viewportIndex, byteArray } = ownProps;
+  const { activeViewportIndex } = state.viewports;
+
+  return {
+    viewportIndex,
+    activeViewportIndex,
+    byteArray,
```

Comment:
advantage of mapping `byteArray` to `byteArray`?

Reply:
Not at all, removed. ⭐️